orttraining/orttraining/runner/orttraining_apis.h

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Copyright (c) SignalPop LLC.  All rights reserved.
+// Licensed under the MIT License.
+
+namespace OrtTrainingApis {
+
+ORT_API(const OrtTrainingApi*, GetApi, uint32_t version);
+ORT_API(const char*, GetVersionString);
+
+ORT_API(void, ReleaseTrainingParameters, _Frees_ptr_opt_ OrtTrainingParameters*);
+
+ORT_API_STATUS_IMPL(CreateTrainingParameters, _Outptr_ OrtTrainingParameters** out);
```

Comment:
>ORT_API_STATUS_IMPL [](start = 0, length = 19)

This macro is not used for protos.

Reply:
I trying to follow the way it is done in the following: https://github.com/MyCaffe/onnxruntime/blob/WinTrainingCSharp/onnxruntime/core/session/ort_apis.h 

For example, on line 29:
ORT_API_STATUS_IMPL(CreateEnv, OrtLoggingLevel logging_level, _In_ const char* logid, _Outptr_ OrtEnv** out)
ORT_ALL_ARGS_NONNULL